src/lib/MedStickerEncryptionV2.js

```diff
@@ -0,0 +1,85 @@
+import create from "./factory";
```

Comment:
Now we have 3 Med Sticker versions,

Version 1 and 2 named A and B implemented in `MedStickerEncrypted.js`

And Version 3 named C and implemented in `MedStickerEncryptedV2.js` 

This is really confusing

Reply:
Why arent we doing this through the same API as the first 2 versions?

Reply:
@Weetbix you mean that these all should be living in the same directory or?
I agree that naming can be better, any suggestions?

Reply:
The suggestion was to have a separate charlie class

Reply:
@Weetbix its not fully compatible with the old api even on signatures level, not speaking about implementation

Reply:
v3 is living in a v2 file... would be better if instead you stick to the naming (and keep the version correct) like `medStickerCharlie.js` 

I dont mind if the implementations are in separate files, but currently you would have to import decrypt from 2 different places and know what version you want. 

Why arent we updating this: https://github.com/VivyTeam/krypt-web/blob/master/src/lib/MedStickerEncryption.js#L23

Reply:
@Weetbix iv now comes from the server, besides the qr code content secret from the backend is now required. I can introduce you new security concept to give you more context

Reply:
Have we concluded that this file should be renamed MedStickerEncryptionV1.js?

Reply:
This should be named `medStickerEncryptionCharlie` 

I would like to know how you plan to expose this API publically too, what the interface will look like? as in this PR its not usable, as its not exported in https://github.com/VivyTeam/krypt-web/blob/master/src/index.js